gateway/api/views/jobs.py

```diff
@@ -145,6 +154,65 @@ def list(self, request):
             serializer = self.get_serializer_job_without_result(queryset, many=True)
         return Response(serializer.data)
 
+    @action(methods=["GET"], detail=False, url_path="provider")
+    def provider_list(self, request):
```

Comment:
Remember @paaragon that you need to include this new end-point in the v1 too:
https://github.com/Qiskit/qiskit-serverless/blob/main/gateway/api/v1/views/jobs.py

Add to it the swagger configuration also.